app/presenters/track.rb

```diff
@@ -47,17 +47,17 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        File.
-          read("./x/docs/md/track/BETTER.md").
-          gsub('REPO', trackler_track.repository).
-          gsub('TOPIC', topic_key.to_s.upcase).
-          gsub('EXT', trackler_track.doc_format)
+        content_with_trackler_data("./x/docs/md/track/BETTER.md").gsub('TOPIC', topic_key.to_s.upcase)
       end
 
       def fallback_topic_content(topic_key)
         filepath = "./x/docs/md/track/#{topic_key.upcase}.md"
         return '' unless File.exist?(filepath)
-        File.read(filepath)
+        content_with_trackler_data(filepath)
+      end
+
+      def content_with_trackler_data(filepath)
+        File.read(filepath).gsub('REPO', trackler_track.repository).gsub('EXT', trackler_track.doc_format)
```

Comment:
More good points!

In https://github.com/exercism/exercism.io/pull/3475/commits/33cdb6812164e186d88287c83359fa81bb032204, I made sure that we were reading the `File` in the same method we check for its existence.

In https://github.com/exercism/exercism.io/pull/3475/commits/2b344986e63e780cffe60f6a73e52a04ed0b91ad, I made some naming changes for clarity. I like how your suggestion of "template" signals content that has parts that still need to be filled in, so I latched on to that.